prow/config/jobs/proxy-1.10.yaml

```diff
@@ -0,0 +1,231 @@
+branches:
+- release-1.10
+image: gcr.io/istio-testing/build-tools-proxy:release-1.10-2021-04-02T23-15-48
+jobs:
+- command:
+  - ./prow/proxy-presubmit.sh
+  name: test
+  node_selector:
+    testing: build-pool
+  requirements:
+  - cache
+  - gcp
+  timeout: 4h0m0s
+  types:
+  - presubmit
+- command:
+  - ./prow/proxy-presubmit-asan.sh
+  name: test-asan
+  node_selector:
+    testing: build-pool
+  requirements:
+  - cache
+  - gcp
+  timeout: 4h0m0s
+  types:
+  - presubmit
+- command:
+  - ./prow/proxy-presubmit-tsan.sh
+  name: test-tsan
+  node_selector:
+    testing: build-pool
+  requirements:
+  - cache
+  - gcp
+  timeout: 4h0m0s
+  types:
+  - presubmit
+- command:
+  - ./prow/proxy-presubmit-release.sh
+  name: release-test
+  node_selector:
+    testing: build-pool
+  requirements:
+  - cache
+  - gcp
+  timeout: 4h0m0s
+  types:
+  - presubmit
+- command:
+  - ./prow/proxy-presubmit-centos-release.sh
+  image: gcr.io/istio-testing/build-tools-centos:master-2021-04-02T23-15-48
```

Comment:
this looks wrong

Reply:
Agree. Looks like generate.go only changes the outer image, and not the inner images, or there is something changing it to master. Adding a note to investigate the generate.go and will fix manually.